### ISSUE TITLE
fix main loop browser timing discrepancies

### ISSUE THREAD

**Issue by pancelor** [Auroriax/PuzzleScriptPlus#81]
Hi there! I ran my tests from https://github.com/increpare/PuzzleScript/issues/949 and the browser discrepancy is present here in PSP too -- chrome runs key_repeat_interval about 30% slower than firefox does, at least on my machine.

I haven't tested my fix (issue 948, linked in 949 above), but I expect it will fix things here too. I've made a few games in PSP since learning about the issue, and I manually merge my fix into the html every time I export -- annoying! it's not the worst thing in the world, but it'd be nice to have it merged in this repo. I can make the PR if you'd like, I'm just asking first to see if you're open to it

**Comment by Auroriax** [comment]
Yes, please make a PR! I've noticed https://github.com/increpare/PuzzleScript/issues/949 at some point (and the related discussion in the Thinky Puzzle Games Discord server), and it looks useful! I was waiting for it to be merged into vanilla PS, but since I'm still a bit behind on merging those (#23), this seems like a good idea!
